src/content/ui-api/observability/grafana/custom-dashboards/index.md

```diff
@@ -0,0 +1,57 @@
+---
+linkTitle: Custom Grafana dashboards
+title: Creating custom Grafana dashboards
+description: Guide explaining how to manage custom Grafana dashboards in our management cluster Grafana.
+menu:
+  main:
+    identifier: uiapi-observability-grafana-custom-dashboards
+    parent: uiapi-observability-grafana
+weight: 40
+aliases:
+  - /observability/grafana/custom-dashboards
+last_review_date: 2022-09-12
+user_questions:
+  - How can customize dashboards?
+  - How can I create my own dashboards?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+__Beware__ that the access to this feature is experimental and the implementation specifics can change at a later time (e.g. in favor of Grafana Operator)
+
+As explained in our guide explaining [how to access Grafana]({{< relref "/ui-api/observability/grafana/access" >}}), we provide a subset of dashboards accessible by our customers.
+
+Sometimes, the dashboards we provide are not enough for your use case so we offer you the capability of creating your own (e.g. you are deploying your own apps on the management cluster, or you want custom dashboards for your clusters).
+
+## Creating your own dashboard
+
+To create your own dashboard, you can create a configmap in the management cluster in any namespace you want (preferably the one you use for automation) and create a configmap containing the dashboard. For example:
+
+```yaml
+apiVersion: v1
+data:
+  my-dashboard.json: |-
+    { ... my dashboard in json format }
+kind: ConfigMap
+metadata:
+  annotations:
+    ## Define the directory in grafana where the dashboard will be added to the grafana container
+    k8s-sidecar-target-directory: /var/lib/grafana/dashboards/customer
+  labels:
+    ## Tell grafana to load this configmap as a dashboard
+    app.giantswarm.io/kind: dashboard
+  name: my-grafana-dashboard
+  namespace: my-namespace
+```
+
+### Limitations
+
+__Beware__ that the dashboard name must be unique so do not override one of your own.
```

Comment:
is there a chance it overrides one of ours?

Reply:
No because they are not in the same folder but there is nothing preventing them from setting our folder instead. We trust our users are not dumb because there is no way to enforce that limitation without reallllly hard to implement webhooks